stdlib/public/core/Unmanaged.swift

```diff
@@ -219,6 +221,11 @@ public struct Unmanaged<Instance: AnyObject> {
   }
 
   /// Performs an unbalanced retain of the object.
+  ///
+  /// Note: Use Umanaged.passRetained(object) instead to ensure that
```

Comment:
Hm, I think this note is confusing/misleading. IIUC, it tries to warn against invocation chains such as `passUnretained(foo).retain()` -- however, those should already look wrong as is!

`passRetained` is not a substitute for the `retain` method; suggesting it as such looks nonsensical/confusing.

Reply:
@lorentey I'll put up a new PR removing that comment. I just want to be sure I understand: you're not worried about users of the Unmanaged API making the same mistake that the stdlib author made when implementing `passRetained`?